src/SearchField/index.jsx

```diff
@@ -158,9 +163,9 @@ SearchField.defaultProps = {
   formAriaLabel: undefined,
   value: '',
   screenReaderText: {
-    label: 'search',
-    submitButton: 'submit search',
-    clearButton: 'clear search',
+    label: SEARCH_FIELD_SCREEN_READER_TEXT_LABEL,
+    submitButton: SEARCH_FIELD_SCREEN_READER_TEXT_SUBMIT_BUTTON,
+    clearButton: SEARCH_FIELD_SCREEN_READER_TEXT_CLEAR_BUTTON,
```

Comment:
```suggestion
    label: SEARCH_FIELD_SCREEN_READER_LABEL_TEXT,
    submitButton: SEARCH_FIELD_SCREEN_READER_SUBMIT_BUTTON_TEXT,
    clearButton: SEARCH_FIELD_SCREEN_READER_CLEAR_BUTTON_TEXT,
```
I think it is cleaner to have `ELEMENT_ATTRIBUTE` naming instead of `ATTRIBUTE_ELEMENT` like in the BEM for `css`